test/index.js

```diff
@@ -25,6 +25,13 @@ const expect = Code.expect;
 
 describe('Bananas', () => {
 
+    const originalPost = Wreck.post;
+    lab.afterEach((done) => {
```

Comment:
why? 

There was a real issue here - some of the tests were returning `Wreck.post` to its original form before the tests were done (causing real network requests to loggly api), so I thought it's a good candidate for afterEach - otherwise you have to write it down every time you put done.

Usually, I use afterEach when I have 'fake' tests. That way I can ensure whatever the test did, It got cleaned. The afterEach here is 'unobtrusive' - no harm done if you run it multiple times, it's just stating the fact `Wreck.post` returns to its original form.

I guess the `afterEach` can cause some weird feeling of magic, because you don't see what's going on directly on the test you write